airbyte/manifest.json

```diff
@@ -0,0 +1,37 @@
+{
+  "manifest_version": "2.0.0",
+  "app_uuid": "5994a02c-8754-40c3-9e99-a39ffc862b1c",
+  "app_id": "airbyte",
+  "display_on_public_website": false,
```

Comment:
we may want to enable it directly?

Reply:
I was also thinking about that, good to have a second opinion. I'll enable it.